src/page-templates/content-page/content-page-template.tsx

```diff
@@ -364,81 +375,186 @@ const ContentPageTemplate: NextPage<ContentPageProps> = ({
             2,
             -1
         );
+        let displayHeaderImage = true;
+        // Industry Events do not have to show a placeholder image
+        if (isIndustryEvent && !image?.url) {
```

Comment:
Can this just be `const displayHeaderImage = !industryEvent || image?.url`?

Reply:
also maybe I'm missing something, but is the logic here that images on industry events are optional, or that they are never shown no matter what? if it's the former, then it seems that we're missing some lines in the gql to grab the image field. the reason i want to confirm this logic is because i tested on my local and navigated to an industry event that had an image in strapi, but no image showed up on the industry event page.

Reply:
ah crap good catch, I missed adding it when I was merging our two queries together